tests/unit/Codeception/Module/FacebookTest.php

```diff
@@ -11,12 +11,13 @@
 class FacebookTest extends \PHPUnit_Framework_TestCase
 {
     protected $config = array(
-        'app_id'    => '460287924057084',
-        'secret'    => 'e27a5a07f9f07f52682d61dd69b716b5',
+        'app_id' => '1584417941835888',
+        'secret' => '17e014121a007dc5d149ada7f9201d07',
         'test_user' => array(
             'permissions' => array(
-                'publish_stream'
-            )
+                'public_profile',
```

Comment:
There's no need for any special permission for it. Public profile is there by default in v4 API.